### PR TITLE
Adding questions

### PR DESCRIPTION
**Pull request by scove-viber**
## What type of PR is this? (check all applicable)

- [ ] 🚀 Added Name
- [ ] ✨ Feature
- [ ] 🌟 stared the repo
- [ ] 🐛 Grammatical Error
- [ ] 📝 Documentation Update
- [ ] 🚩 Other

### CODE REVIEW

File: Languages/DSA/questions.json [hunk 1]
```diff
@@ -98,6 +98,33 @@
...
       "shortAns": "It is an efficient algorithm to search an element in a sorted list",
       "tag": "binary-search"
+    },
+    {
+      "id": 11,
+      "question": "Minimum number of queues required for priority queue implementation?",
+      "details": "Minimum number of queues required for priority queue implementation is two. One for storing actual data and one for storing priorities.",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "Atleast two queues requried for priority queue implementation.",
```
Comment (Tc2r1) [inline Languages/DSA/questions.json:108]: Please follow the correct syntax at "shortAns" 

shortAns strings should begin with "It is" and should not have a reference to the words used in the question as this makes it obvious that that is the correct answer. 

When the quiz is built, each question will show with 4 different "short answers" if one of the short answers has the same wording, it becomes obvious that is the correct answer.

File: Languages/DSA/questions.json [hunk 2]
```diff
@@ -98,6 +98,33 @@
...
+      "tag": "queues"
+    },
+     {
+      "id": 12,
+      "question": "What is a graph?",
+      "details": "A graph is a non-linear data structure, which consists of vertices (or nodes) connected by edges (or arcs) where edges may be directed or undirected. In Computer science graphs are used to represent the flow of computation..",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "A graph is a pictorial representation of a set of objects where some pairs of objects are connected by links. The interconnected objects are represented by points termed as vertices, and the links that connect the vertices are called edges.",
```
Comment (Tc2r1) [inline Languages/DSA/questions.json:117]: Again, please dont do "What is a cat" and then " A cat is" short answers should not use the same terminology as used in the question.

File: Languages/DSA/questions.json [hunk 3]
```diff
@@ -98,6 +98,33 @@
...
+      "tag": "graph"
+    },
+     {
+      "id": 13,
+      "question": "How Prim's algorithm finds spanning tree?",
+      "details": "Prim's algorithm to find minimum cost spanning tree (as Kruskal's algorithm) uses the greedy approach. Prim's algorithm shares a similarity with the shortest path first algorithms. Prim's algorithm, in contrast with Kruskal's algorithm, treats the nodes as a single tree and keeps on adding new nodes to the spanning tree from the given graph.",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "A graph is a pictorial representation of a set of objects where some pairs of objects are connected by links. The interconnected objects are represented by points termed as vertices, and the links that connect the vertices are called edges.",
```
Comment (Tc2r1) [inline Languages/DSA/questions.json:126]: "shortAns" is short for "Short Answer" implying that it should be short, please try to keep the short answers short. 2 of yours are way too long. Thanks

**Review by Tc2r1 — CHANGES_REQUESTED** [review]

File: Languages/DSA/questions.json (hunk 1 above)
Comment (ShrinathPawar) [reply Languages/DSA/questions.json:108]: Ok so the short answer here should be "it's two" or something like "two queues only", right?

File: Languages/DSA/questions.json (hunk 2 above)
Comment (ShrinathPawar) [reply Languages/DSA/questions.json:117]: Now here do i need to change in both detail answer and short answer?
I need to start the answer from "It is" but not with "The graph is"...

File: Languages/DSA/questions.json (hunk 3 above)
Comment (ShrinathPawar) [reply Languages/DSA/questions.json:126]: Ya i got it .
Last two questions are having a bit long answers which has to be short.

File: Languages/DSA/questions.json (hunk 1 above)
Comment (Tc2r1) [reply Languages/DSA/questions.json:108]: I would re-ask the question in a way that doesn't have such a direct answer. like "what is a priority queue implementation? "

File: Languages/DSA/questions.json (hunk 2 above)
Comment (Tc2r1) [reply Languages/DSA/questions.json:117]: Only need to change the shortAns, look at the Android json files for reference.

File: Languages/DSA/questions.json (hunk 3 above)
Comment (Tc2r1) [reply Languages/DSA/questions.json:126]: @ShrinathPawar will you be making changes?

Comment (ShrinathPawar) [reply Languages/DSA/questions.json:126]: Yep , by today I'll get it done ...

Comment (Tc2r1) [reply Languages/DSA/questions.json:126]: @ShrinathPawar  you didn't get it done? :(

**Comment by Tc2r1** [comment]
due to lack of response and detailed handling of changes, this PR is being labelled invalid and closed unfortunately.

**Pull request closed without merging**